czkawka-cli: add page (#9477)
tldr-pages/tldr: pages/common/czkawka-cli.md

# czkawka-cli

> > Multi functional app to find duplicates, empty folders, similar images etc. Command-line version.

Reviewer: ```suggestion
> Command-line version of `czkawka` a multi-functional app to find duplicates, empty folders, similar images and much more.
```
Would this be better?

> More information: <https://github.com/qarmin/czkawka>.

> - Find [dup]licate files in directory:

Reviewer: ```suggestion
- Find duplicate files in a specific directory:
```
Would this be better?

> - Find [dup]licate files in directory:
> 
> `czkawka-cli dup --directories {{path/to/directory}}`

Reviewer 2: ```suggestion
- List duplicate or similar files in specific directories:

`czkawka-cli {{dup|image}} --directories {{path/to/directory1 path/to/directory2 ...}}`
```

- Optional fix (multiple occurrences): `list` may sound better as it's not clear what is going to be with search results with the current description.
- Optional fix (multiple occurrences): `specific` may sound better.
- Optional fix (multiple occurrences): mnemonics can be not used with long options.
- Optional fix: this example can be joined with the last one.
- Optional fix (multiple occurrences): example can be generalized.

> - Find [dup]licate files and delete them (default: `NONE`):

Reviewer: ```suggestion
- Find duplicate files and delete them (default: `NONE`):
```
This isn't necessary here as the command doesn't have long arguments.

> - Find [dup]licate files and delete them (default: `NONE`):
> 
> `czkawka-cli dup --directories {{path/to/directory}} --delete-method {{AEN/AEO/ON/OO/HARD/NONE}}`

Reviewer 2: ```suggestion
- Find duplicate files in specific directories and delete them (default: `NONE`):

`czkawka-cli dup --directories {{path/to/directory1 path/to/directory2 ...}} --delete-method {{AEN|AEO|ON|OO|HARD|NONE}}`
```

- Optional fix: example can be generalized.
- Mandatory fix: [`/` must be replaced with `|`](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases).

> - Find similar [image]s:

Reviewer: ```suggestion
- Find similar images:
```

> - Find similar [image]s:
> 
> `czkawka-cli image --directories {{path/to/directory}}`

Reviewer 2: ```suggestion
```

Optional fix: this example can be joined with the first one.
